Anna_Mayilyan/python/25_05/perfect_num.py

```diff
@@ -0,0 +1,9 @@
+n = int(input(" Enter number: "))
+S = 0
+for i in range(1, n):
+    if(n % i == 0):
+        S = S + i
+if (S == n):
+    print(" %d is a Perfect Number" %n)
+else:
+    print(" %d is not a Perfect Number" %n)
```

Comment:
- No any function
- No any input validation
![image](https://user-images.githubusercontent.com/80816144/119726431-27f00780-be82-11eb-946f-3691522d6dae.png)

- 0 is not perfect number
- variable names :((